### DISCUSSION TITLE
2022-43: 工作中的自我调适

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#385] [Announcements]
# 2022-43: 工作中的自我调适

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-43/

**Comment by abersheeran** [comment]
最后一段话感觉受了重伤……没有爱人🥹

**Comment by lllong33** [comment]
告别宏大叙事, 从简单易行的小事做起, 是一种不错的参考;
